### DISCUSSION TITLE
Issue Starting ZigbeeHerdsman Zigbee2MQTT Sonoff zigbee 3.0 usb dongle plus

### DISCUSSION THREAD

**Discussion by keithcrooks1** [Koenkk/zigbee2mqtt#12742] [Devices]
Hi wasnt sure where to post this issue, I have been using Zigbee2mqtt for ages now no issues, Saturday night boom it just stopped. mesed about for ages and then though ok I will just get a new dongle and setup a new HA install.
PI4 with sonoff zigbee 3.0 usb dongle plus, HA identifies the dongle no issues, I can even add Devices via ZHA
I installed Zigbee2MQTT and Mosquitto broker, Setup Zigbee2MQTT Config as follows:
data_path: /config/zigbee2mqtt
socat:
enabled: false
master: pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777
slave: tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5
options: '-d -d'
log: true
mqtt:
base_topic: zigbee2mqtt
user: mqtt1
password: mqtt1
server: mqtt://localhost
serial:
port: >-
/dev/serial/by-id/usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_600c32e88ac9eb11b0348d4f1d69213e-if00-port0
zigbee_herdsman_debug: false
zigbee_shepherd_devices: false

Error message:
Zigbee2MQTT:info 2022-06-06 23:25:29: Logging to console and directory: '/config/zigbee2mqtt/log/2022-06-06.23-25-28' filename: log.txt
Zigbee2MQTT:info 2022-06-06 23:25:29: Starting Zigbee2MQTT version 1.25.2 (commit #unknown)
Zigbee2MQTT:info 2022-06-06 23:25:29: Starting zigbee-herdsman (0.14.34)
Zigbee2MQTT:error 2022-06-06 23:25:49: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2022-06-06 23:25:49: Failed to start zigbee
Zigbee2MQTT:error 2022-06-06 23:25:49: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2022-06-06 23:25:49: Exiting...
Zigbee2MQTT:error 2022-06-06 23:25:49: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
at ZStackAdapter.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/zStackAdapter.ts:102:27)
at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:123:29)
at Zigbee.start (/app/lib/zigbee.ts:58:27)
at Controller.start (/app/lib/controller.ts:101:27)
at start (/app/index.js:109:5)
[23:25:52] INFO: Handing over control to Zigbee2mqtt Core ...

zigbee2mqtt@1.25.2 start
node index.js

I have run this no issues [core-ssh ~]$ test -w /dev/ttyUSB0 && echo success || echo failure
success
I have tried the donge both on an extension and directly into the Pi4, I have no low voltage or under voltage issues any help please, wife is unhappy with the darkness cheers

**Comment by Nardol** [comment]
I read you tested with ZHA, now have you fully disabled ZHA? By removing it from your Home Assistant configured integrations.
If it is still accessing the port it will prevent Zigbee2MQTT from using it.
You should also disable ZHA when it is auto-discovered with the dongle.

Also in `advanced:` section you could try to set `  rtscts: false`
If I remind well it is the default value but within setting it to false you will be sure of the configuration.
Maybe reflashing your sonoff dongle could help.

**Reply by keithcrooks1** [reply]
cheers ill try that thankyou

**Reply by keithcrooks1** [reply]
Ok so ZHA has been disabled from my system. I have restarted my Pi and the Zigbee2MQTT
Wasnt sure where to find this bit: "Also in advanced: section you could try to set rtscts: false"
![image](https://user-images.githubusercontent.com/5075339/172337911-42d2f8b3-d253-43aa-8a93-f978e18205f7.png)
![image](https://user-images.githubusercontent.com/5075339/172339876-c1d05363-e8e3-4247-9bd7-74368d76bf9b.png)
cheers

**Reply by Nardol** [reply]
At the end of your config, add

```yaml
advanced:
  rtscts: false
```

**Comment by keithcrooks1** [comment]
I have refreshed the stick F/W,
This is my Config attached:
My other HA is running a Pi4 but has no usb stick in it but is all on same network.
I will try another fresh install of HA to my Pi and I may order another stick too just incase
 
![image](https://user-images.githubusercontent.com/5075339/172382667-1a42410e-0143-48b6-abf9-7dfefb6a57b3.png)
Log:
![image](https://user-images.githubusercontent.com/5075339/172382920-ea57d9fc-0478-4717-82da-fba4a1494f3a.png)

**Comment by Nardol** [comment]
See #10209 it looks like your Pan ID or extended pan id must be changed.
And https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html
There is a section about the PAN ID which, I hop, might help you.

**Reply by keithcrooks1** [reply]
cheers ill take a look tonight

**Comment by keithcrooks1** [comment]
Right moving slowly in some direction.
Re-flashed USB Stick
Fresh Install of HA
Inserted USB Stick before power up
Stick was recognised and I chose Ignore
![image](https://user-images.githubusercontent.com/5075339/172481071-59929749-4b5d-41b4-9da1-98f7011cef4d.png)
Installed MQTT Broker as per instructions
Installed Zigbee2MQTT as per instructionss
![image](https://user-images.githubusercontent.com/5075339/172481382-522d93e0-2469-4ad7-a12c-738d57544b72.png)
Zigbee2mqtt yaml:
![image](https://user-images.githubusercontent.com/5075339/172481489-693eb202-a5cf-4b46-b361-99ee247670c8.png)
Log:
![image](https://user-images.githubusercontent.com/5075339/172481713-db0f82ef-a854-423c-8609-663c91a7cad3.png)
I have ordered a new stick just in case, but I just now have no idea
Flashing File:
![image](https://user-images.githubusercontent.com/5075339/172482345-67bfec17-96ef-4395-ac1a-0d2e6c4e8b9d.png)
I may have to give up on this and just pile efforts into Zwave 
HA Version:
![image](https://user-images.githubusercontent.com/5075339/172482545-2a58b1cf-81d7-4029-8a4b-8b37e7600d42.png)

**Reply by msteingr** [reply]
hello did you get it working I am facing the same issue  I am try to switch from my Conbee 2 using zigbee2 mitt but facing issues with my ne Sono 3.0 P getting the same error messages..

what did you do to solve the issue?

**Comment by midasf** [comment]
Your zigbee2mqtt config seems off, can you change the serial part?
`serial:
  port: /dev/serial/by-id/usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_600c32e88ac9eb11b0348d4f1d69213e-if00-port0`

**Comment by keithcrooks1** [comment]
i got a different dongle sent it back and purchased another manufacturers
product

**Comment by msteingr** [comment]
Did you bought another sonoff 3.0 dongle ?
Or which did you buy

**Comment by keithcrooks1** [comment]
i got the
https://www.amazon.co.uk/gp/product/B07PZ7ZHG5/ref=ppx_yo_dt_b_asin_title_o01_s00?ie=UTF8&psc=1
ConBee II The universal Zigbee USB gateway

**Comment by msteingr** [comment]
Interesting I have a conbee 2 at since last firmware it gives me a
headache. Some of my lamps are not switching on groups react strange and I
am loosing endpoints.   Have about 120 devices …
So I had read  that the sonoff is better
But looks like mine sonoff 3 is not working correctly

**Comment by keithcrooks1** [comment]
i never got mine working, my house is fairly small with about 43 devices
all seem to behave mostdays
